vignettes/formats_precedence.Rmd

```diff
@@ -0,0 +1,300 @@
+---
+title: "Formats Precedence and NA Handling"
+author: "Wojciech Wójciak and Gabriel Becker"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Formats Precedence NA Handling}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options: 
+  chunk_output_type: console
+---
+
+```{r, echo=FALSE}
+knitr::opts_chunk$set(comment = "#")
+```
+
+```{css, echo=FALSE}
+.reveal .r code {
+  white-space: pre;
+}
+```
+
+## Formats Precedence
+
+A user of `rtables` package can specify the format in which the numbers in the reporting tables are printed. Formatting functionally is provided by the [`formatters`](https://insightsengineering.github.io/formatters/) R package and available formats can be listed with `formatters::list_valid_format_labels()` function.
+The format can be specified by the user in a few different places. It may happen that for a single table layout, the format is specified in more than one place. In such a case, the final format that will be applied, depends on formats precedence rules defined by `rtables`. In this vignette, we describe basic rules of `rtables` formats precedence.
```

Comment:
```suggestion
The format can be specified by the user in a few different places. It may happen that for a single table layout, the format is specified in more than one place. In such a case, the final format that will be applied depends on formats precedence rules defined by `rtables`. In this vignette, we describe the basic rules of `rtables` formats precedence.
```